content/en/docs/setup/production-environment/dockershim.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Dockershim Deprecation FAQ
+content_type: concept
+aliases: [ '/dockershim']
+weight: 40
+---
+
+This document goes over some frequently asked questions and their answers. For
+more detail on the deprecation of Docker as a container runtime for Kubernetes
```

Comment:
there's already a blog post verison https://kubernetes.io/blog/2020/12/02/dont-panic-kubernetes-and-docker/

Reply:
its being migrated to a blog post now. 👍

Reply:
Yes, I understand. How will readers know to look in this section for important information?
It would be good to broadcast widely and often (release notes?).